07.bowling_object/game.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require_relative 'frame'
+
+class Game
+  STRIKE = 'X'
+
+  def initialize(marks)
+    @marks = marks.split(',')
+  end
+
+  def score
+    frames = create_frames
+    frames.each_with_index.map do |frame, idx|
```

Comment:
map+sumでなく、sumだけでも合計値が出せそうです。

```suggestion
    frames.each_with_index.sum do |frame, idx|
```